src/components/ResourceItem/tests/ResourceItem.test.tsx

```diff
@@ -310,106 +320,125 @@ describe('<ResourceItem />', () => {
         </ResourceListContext.Provider>,
       );
 
-      findByTestID(wrapper, 'Item-Wrapper').simulate('click');
+      wrapper
+        .find('div', {className: styles.ResourceItem})!
+        .trigger('onClick', {
+          stopPropagation: () => {},
+          nativeEvent: {},
+        });
       expect(onClick).toHaveBeenCalledWith(itemId);
     });
 
     it('calls window.open on metaKey + click', () => {
-      const wrapper = mountWithAppProvider(
+      const wrapper = mountWithApp(
         <ResourceListContext.Provider value={mockDefaultContext}>
           <ResourceItem id={itemId} url={url} accessibilityLabel={ariaLabel} />
         </ResourceListContext.Provider>,
       );
-      const item = findByTestID(wrapper, 'Item-Wrapper');
-      trigger(item, 'onClick', {
-        stopPropagation: () => {},
-        nativeEvent: {metaKey: true},
-      });
+
+      wrapper
+        .find('div', {className: styles.ResourceItem})!
+        .trigger('onClick', {
+          stopPropagation: () => {},
+          nativeEvent: {metaKey: true},
+        });
       expect(spy).toHaveBeenCalledWith(url, '_blank');
     });
 
     it('calls onClick when hitting keyUp on the item when onClick and URL exists', () => {
       const onClick = jest.fn();
-      const wrapper = mountWithAppProvider(
+      const wrapper = mountWithApp(
         <ResourceListContext.Provider value={mockDefaultContext}>
           <ResourceItem id={itemId} url="#" onClick={onClick} />
         </ResourceListContext.Provider>,
       );
 
-      findByTestID(wrapper, 'Item-Wrapper').simulate('keyup', {
-        key: 'Enter',
-      });
+      wrapper
+        .find('div', {className: styles.ResourceItem})!
+        .trigger('onKeyUp', {key: 'Enter'});
 
       expect(onClick).toHaveBeenCalled();
     });
 
     it('does not call onClick when hitting keyUp on non Enter key', () => {
       const onClick = jest.fn();
-      const wrapper = mountWithAppProvider(
+      const wrapper = mountWithApp(
         <ResourceListContext.Provider value={mockDefaultContext}>
           <ResourceItem id={itemId} url="#" onClick={onClick} />
         </ResourceListContext.Provider>,
       );
 
-      findByTestID(wrapper, 'Item-Wrapper').simulate('keyup', {
-        key: 'Tab',
-      });
+      wrapper
+        .find('div', {className: styles.ResourceItem})!
+        .trigger('onKeyUp', {key: 'Tab'});
 
       expect(onClick).not.toHaveBeenCalled();
     });
 
     it('does not call onClick when hitting keyUp on the item when no URL exists', () => {
       const onClick = jest.fn();
-      const wrapper = mountWithAppProvider(
+      const wrapper = mountWithApp(
```

Comment:
I'm wondering if we should move it out of the default group or change it's context to the default one, since the test itself seems related to its neighbours, it's just the context that's different (which could have been some autocomplete mistake). What do you think?